README.md

```diff
@@ -1,76 +1,71 @@
+- [Security Guidelines](./SECURITY.md)
+- [Vision](./VISION.md)
+- [Roadmap Tracker](https://github.com/orgs/openebs/projects/78)
 
-Meeting Link: <https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1> <BR>
-Starting from August 2024, the meetings are recorded and posted on YouTube: <https://www.youtube.com/@openebscommunity6021> <BR>
+## Project Leadership team
 
-## Code of Conduct
+This Community is managed by the OpenEBS [Maintainers](./MAINTAINERS.md) within the OpenEBS project team. We liaise with the Linux Foundation and CNCF on project, governance
+and operational matters. We curate the operations of the project, product, roadmaps, initiatives, all engineering/code activities and all events (including conferences).
 
-OpenEBS follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+ [<img alt="Container Storage Interface group" src="images/CNCF_csi-horizontal-color_2024.png" width="320">](https://github.com/kubernetes/community/tree/HEAD/sig-storage)  [<img alt="Storage Technical Advisory Group" src="images/CNCF_tag-storage-horizontal-color_2024.png" width="320">](https://github.com/cncf/tag-storage)  &emsp; &emsp; [<img alt="Cloud Native Community Groups" src="images/CNCF_cncg-icon-color_2024.png" width="200">](https://github.com/cncf/communitygroups)
 
----
-[![OpenEBS Welcome Banner](/images/community_banner_retro_gamer_level-up-2024_transp.png)](https://www.openebs.io/)
+## How can I get involved?
 
-> [!NOTE]
-> CNCF Project status: SANDBOX <BR>
-> ```⚠️``` &nbsp; In Jan 2024, under the guidance of CNCF TOC, our project moved from Sandbox level to Archive level, enabling us to implement an extensive restructuring initiative. We are actively working to archive legacy sub projects, repositories and dependencies to a CNCF owned GitHub organization called: [OpenEBS-Archive](https://github.com/openebs-archive), plus updating our governance and operating procedures.<BR>
-> ```⚠️``` &nbsp; On 13 June 2024, under guidance of CNCF TOC, our project re-Applied for CNCF promotion from ```Archive``` up to ```Sandbox``` level and was accepted back into
-```Sandbox``` in Oct 2024. See our Sandbox promotion [application here](https://github.com/cncf/sandbox/issues/104)
->
-> * What's changing in the technology stack? The product team is unifying ``` Five ``` major Storage Engines (<kbd>Mayastor</kbd>, <kbd>LocalPV-LVM</kbd>, <kbd>LocalPV-ZFS</kbd>, <kbd>LocalPV-Hostpath</kbd>, <kbd>RawFile</kbd>) into one core OpenEBS code-base.
-> * For more information, see [the parent issue]( https://github.com/openebs/openebs/issues/3701) in our main repo for more detail.
->
-<BR>
+One of the easiest ways to contribute is to participate in discussions at community engagements.
 
----
+If you're looking for something to work on, read the [contribution guidelines](./CONTRIBUTING.md) and then you start by looking for GitHub issues, marked with `kind/good-first` or the `kind/help-wanted` labels.
```

Comment:
I think these two labels are named 'good first issue' and 'help wanted' respectively.

Reply:
Let's leave out the label discussion out of this PR. I will raise a different one once we all decide.